mods/cnc/maps/cnc64gdi01/map.yaml

```diff
@@ -731,6 +731,23 @@ Actors:
 Smudges:
 
 Rules:
+	^Palettes:
```

Comment:
It might be better to have a `^PlayerPalettes` trait set and inherit that as well where `^Palettes` is inherited instead of redefining in each `map.yaml`.